src/fmu/dataio/_definitions.py

```diff
@@ -32,6 +33,7 @@ def __post_init__(self):
             "csv|xtgeo": ".csv",  # use default xtgeo columns: X_UTME, Y_UTMN, Z_TVDSS
             "irap_ascii": ".poi",
         }
+        self.dictionary = {"json": ".json", "yaml": ".yml"}
```

Comment:
Good example of something that will be potentially very difficult to understand 6 months from now, without the context. If we move on with the `dictionary` as a class, we probably need to explain this with comments. If we have no better options, we will have to go with it, but this is a major code smell I think.